### ISSUE TITLE
Accessibility of browse nearby

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#462]
Needs to alert SR to open/collapse behaviour. TBD

**Comment by aeschylus** [comment]
How does this relate to the conversation we had Jennifer (removing side buttons and enabling horizontal scroll)? Or is that a different ticket (mobile accessibility)?

**Comment by jvine** [comment]
Separate issue. We have a broad issue of content that is exposed or hidden by a click - in browse nearby, in the summary/at the library etc. controls on the search results, and in previews - we are not informing the screen reader that there is new content. There are separate tickets for these, but there should be a shared pattern to resolve them. 

The other conversation we had was about the general interaction of the browse nearby widget for mobile and non-mobile users alike - it's fussy and doesn't use native controls.
